examples/http-rs/README.md

```diff
@@ -0,0 +1,37 @@
+# Simple HTTP Rust Server on Unikraft
```

Comment:
Update the `README.md` file to feature similar contents to the `README.md` file from PR #97 .

Reply:
Done. I have not yet looked at running this with `kraft` and networking, though.